packages/next/src/server/config.ts

```diff
@@ -1476,3 +1478,22 @@ export function getConfiguredExperimentalFeatures(
   }
   return configuredExperimentalFeatures
 }
+
+function cloneObject(obj: any): any {
```

Comment:
We gave to build a customized version, structureClone also has limitation cause function can't be cloned with it.